simplq/src/components/pages/Join/Form.jsx

```diff
@@ -75,7 +75,7 @@ export function JoinQueueForm(props) {
           <LoadingIndicator />
         </div>
       );
-    if (props.buttonName === 'Add') return <AddToQueueButton onClick={handleClick} />;
+    if (props.buttonName === 'Add') return <Button onClick={handleClick}>Add to Queue</Button>;
```

Comment:
what I meant was,
```
return <JoinQueueButton onClick={handleClick}>{props.buttonName}</Button>
```

no need of an if/else anywhere.

Reply:
I'm not keen on this idea. Then the whole joinQueueButton is just a pointless wrapper.

Reply:
ok. then lets remove that. `return <Button onClick={handleClick}>{props.buttonName}</Button>;`

I want to avoid this `if (props.buttonName === 'Add') ... else ...` thing

Reply:
can we then remove any other unwanted button wrappers and use Button/StandardButton in it's place?

Reply:
I've removed the joinQueueButton since it's not used much anymore. We'll remove the other as required.